src/analysis/normed_space/lattice_ordered_group.lean

```diff
@@ -49,6 +49,11 @@ class normed_lattice_add_comm_group (α : Type*)
 lemma solid {α : Type*} [normed_lattice_add_comm_group α] {a b : α} (h : |a| ≤ |b|) : ∥a∥ ≤ ∥b∥ :=
 normed_lattice_add_comm_group.solid a b h
 
+noncomputable instance : normed_lattice_add_comm_group ℝ :=
+{ add_le_add_left := by finish,
+  solid := by finish,
+  ..real.lattice,
+  ..real.normed_group, }
```

Comment:
You can drop the last two lines, Lean can figure it out. You can also give explicit short proofs of the two first (I wrote `by tidy?` to get suggestions);
```suggestion
{ add_le_add_left := λ _ _ h _, add_le_add le_rfl h,
  solid := λ _ _, id, }
```